src/Scripting/CSharpTest/ScriptTests.cs

```diff
@@ -687,6 +687,35 @@ public void TestCreateMethodDelegate()
             Assert.Equal(0, result);
         }
 
+        [Fact]
+        public async Task ScriptOptionsConfiguredForDebuggingResultInPdbEmitted()
+        {
+            try
+            {
+                var opts = ScriptOptions.Default.WithDebugInformation();
+                var script = await CSharpScript.RunAsync("throw new System.Exception();", opts);
+            }
+            catch (Exception ex)
+            {
+                // line information is only available when PDBs have been emitted
+                Assert.Contains("at Submission#0.<<Initialize>>d__0.MoveNext() in :line 1", ex.StackTrace);
+            }
+        }
+
+        [Fact]
+        public async Task ScriptOptionsNotConfiguredForDebuggingResultInPdbNotEmitted()
+        {
+            try
+            {
+                var script = await CSharpScript.RunAsync("throw new System.Exception();", ScriptOptions.Default);
+            }
+            catch (Exception ex)
+            {
+                // line information is only available when PDBs have been emitted
+                Assert.DoesNotContain("at Submission#0.<<Initialize>>d__0.MoveNext() in :line 1", ex.StackTrace);
```

Comment:
>:line 1 [](start = 88, length = 7)

This is odd, why do we have line number info here?

It would also be good to set the FileName and validate it's included in the stack frame string.